src/services/activityReports.test.js

```diff
@@ -79,9 +79,13 @@ describe('Activity Reports DB service', () => {
   });
 
   afterAll(async () => {
-    await NextStep.destroy({ where: {} });
-    await ActivityRecipient.destroy({ where: {} });
-    await ActivityReport.destroy({ where: {} });
+    const reports = await ActivityReport
+      .findAll({ where: { userId: [mockUser.id, mockUserTwo.id] } });
+    await reports.forEach(async (report) => {
```

Comment:
Async calls inside a forEach can get goofy (see [this post](https://codeburst.io/javascript-async-await-with-foreach-b6ba62bbf404))... I'd probably switch this to a `for (const report of reports)` loop to make sure  next steps and recipients are destroyed before the activity report is destroyed

Reply:
I decided to get rid of the loop altogether and just use a list of id's  in the where clause.